docs/sources/flow/config-language/syntax.md

```diff
@@ -13,56 +13,51 @@ weight: 200
-As such, the ordering of blocks and attributes within the River configuration
-file is not important; the language will consider all direct and indirect
-dependencies between elements to determine their relationships.
+The order of blocks and attributes within the River configuration file isn't important.
+The language considers all direct and indirect dependencies between elements to determine their relationships.
 
 ## Comments
 
-River configuration files support single-line `//` as well as block `/* */`
-comments.
+River configuration files support single-line `//` and block `/* */` comments.
 
 ## Identifiers
 
-River considers an identifier as valid if it consists of one or more UTF-8
-letters (A through Z, both upper- and lower-case), digits or underscores, but
-doesn't start with a digit.
+River considers an identifier as valid if it consists of one or more UTF-8 letters (A through Z, both upper- and lower-case),
+digits or underscores, but doesn't start with a digit.
 
 ## Attributes and Blocks
 
 ### Attributes
 
-_Attributes_ are used to configure individual settings. They always take the
-form of `ATTRIBUTE_NAME = ATTRIBUTE_VALUE`. They can appear either as
-top-level elements or nested within blocks.
+You use _Attributes_ to configure individual settings.
+They always take the form of `ATTRIBUTE_NAME = ATTRIBUTE_VALUE`.
+They can appear either as top-level elements or nested within blocks.
 
 The following example sets the `log_level` attribute to `"debug"`.
 
 ```river
 log_level = "debug"
 ```
 
-The `ATTRIBUTE_NAME` must be a valid River [identifier](#identifiers).
+The `ATTRIBUTE_NAME` must be a valid River [identifier][].
 
-The `ATTRIBUTE_VALUE` can be either a constant value of a valid River
-[type]({{< relref "./expressions/types_and_values.md" >}}) (eg. string,
-boolean, number) or an [_expression_]({{< relref "./expressions/_index.md" >}})
-to represent or compute more complex attribute values.
+The `ATTRIBUTE_VALUE` can be either a constant value of a valid River [type][] (for example, a string, boolean, number),
+or an [_expression_][expression] to represent or compute more complex attribute values.
 
 ### Blocks
 
-_Blocks_ are used to configure the {{< param "PRODUCT_ROOT_NAME" >}}'s behavior as well as {{< param "PRODUCT_NAME" >}} components by
-grouping any number of attributes or nested blocks using curly braces.
-Blocks have a _name_, an optional _label_ and a body that contains any number
-of arguments and nested unlabeled blocks.
+You use _Blocks_ to configure the {{< param "PRODUCT_ROOT_NAME" >}}'s behavior as well as {{< param "PRODUCT_NAME" >}}
+components by grouping any number of attributes or nested blocks using curly braces.
+Blocks have a _name_, an optional _label_ and a body that contains any number of arguments and nested unlabeled blocks.
 
-Some blocks support being defined more than once.
+Some blocks support can be defined more than once.
```

Comment:
This verbiage feels off to me. Removing the word `support` or keeping the original verbiage?

Reply:
Ahhh... you're spot on. I meant to remove `support`